app/src/processing/app/Base.java

```diff
@@ -921,7 +922,10 @@ public boolean handleClose(Editor editor) {
       storeSketches();
 
       // This will store the sketch count as zero
+      editor.setVisible(false);
+      //editor.dispose();
```

Comment:
Let's remove the commented code :)

Reply:
I left it there for a reason :) On my trials, if dispose() gets called the toolbar doesn't disappear but clicking on any element has no effect. I believe it's a broader bug, since the toolbar handlers are declared inside the first Editor instance. If we refactor the code dispose() should be called, so it's more like a reminder :)